ray-operator/api/v1alpha1/raycluster_types.go

```diff
@@ -101,10 +101,8 @@ const (
 )
 
 // RayCluster is the Schema for the RayClusters API
-// +genclient
```

Comment:
We can replace the `+k8s:deepcopy-gen:interfaces=...` marker (which is deprecated in kubebuilder) with `+kubebuilder:object:root=true.`

We don’t need the following markers any more (they’re not used anymore, and are relics from much older versions of KubeBuilder):

```
// +genclient
// +k8s:openapi-gen=true
```

source: https://book.kubebuilder.io/migration/legacy/migration_guide_v1tov2.html